app/src/main/java/ru/prometeydev/movie/data/domain/ActorsDataSource.kt

```diff
@@ -4,10 +4,10 @@ import ru.prometeydev.movie.R
 import ru.prometeydev.movie.data.models.Actor
 
 class ActorsDataSource {
-    fun getActors(): List<Actor> = listOf(
-        Actor("Robert Downey Jr.", R.drawable.actor1),
-        Actor("Chris Evans", R.drawable.actor2),
-        Actor("Mark Ruffalo", R.drawable.actor3),
-        Actor("Chris Hemsworth", R.drawable.actor4)
+    fun getActors(): ArrayList<Actor> = arrayListOf(
```

Comment:
Why array list?

Reply:
to transfer (putParcelableArrayList) through the bundle, but I'll fix it using the movie id when transferring